third_party/2and3/typing_extensions.pyi

```diff
@@ -0,0 +1,32 @@
+import sys
+import typing
+from typing import (
+    ClassVar, ContextManager, Counter, DefaultDict, Deque,
```

Comment:
Really? This stub seemed to work when I tried using it with mypy -- all the types seemed to be correctly re-exported and I was able to successfully my test files.